Adding Raspberry Pi compatible images

plsnotracking opened the pull request:
https://github.com/victor-rds/docker-etesync-server/issues/27

victor-rds commented:
Thanks for the issue and PR.

There is no need for new tags, we can push multiple architectures on the same tag, so instead  `tags/alpine-rpi-32v7/Dockerfile` the file should be `tags/alpine/Dockerfile.arm32v6`, also please use full version numbers on the `FROM`, I keep the images up to date using the dependabot.

I think github action `docker/build-push-action@v1.1.0` doesn't work for ARM images, that's why your PR checks have failed, v2.0 appears to solve this but still unreleased.

TL;DR; Please change the file names, no new tags and remove the workflow files for now, at least until we find whats is causing the errors

plsnotracking commented:
@victor-rds updated, sorry I am not familiar with the github workflows, I can give it a spin over the weekend. Please let me know if you have more feedback. Thanks. Rebased the new commits too.

victor-rds commented:
> @victor-rds updated, sorry I am not familiar with the github workflows, I can give it a spin over the weekend. Please let me know if you have more feedback. Thanks. Rebased the new commits too.

No sweat, I'm no expert in the actions, just noted that your checks failed for no apparent reason, 
If you up to take a look at this https://github.com/docker/build-push-action/pull/92#issuecomment-679094587, maybe we can build multi-arch alpine images without different files just using an workflow file.

Thanks I'm fixing some details of the PGSQL PR, and consider your `new line` nitpick :sweat_smile:

victor-rds commented:
I've tried a new way to build, using a new action, please can you test the ARM images for me? There is new tag `victorrds/etesync:experimental-alpine`, this way I don't need different Dockerfiles for multiple images

plsnotracking commented:
I updated the images to the fix the Dockerfile references (moving them to `/tags/alpine/Dockerfile.<version>` and removed the workflows for now.

If you're talking about  github workflows, I can take a look at them over the weekend.

Am I missing something?

victor-rds commented:
> Am I missing something?

Sorry, I didn't explain everything, there is a new branch called `multiarch`, with a new workflow, I tested and pushed the ARM images do Docker Hub with this tag: [`victorrds/etesync:experimental-alpine`](https://hub.docker.com/r/victorrds/etesync/tags), I can't test them now on my RPi, can you?

plsnotracking commented:
@victor-rds seems to work with the victorrds/etesync:experimental-alpine tag.

My raspi-os is 64bit, would it matter if it was 32 bit?

Thanks.

victor-rds commented:
All tags are build for x64, ARMv6, ARMv7 and ARM64 (AArch64) now, thanks to the new github actions
